ttl/ttlworker/job_manager_integration_test.go

```diff
@@ -180,3 +181,51 @@ func TestTTLAutoAnalyze(t *testing.T) {
 	require.NoError(t, h.Update(is))
 	require.True(t, h.HandleAutoAnalyze(is))
 }
+
+func TestTTLJobDisable(t *testing.T) {
+	failpoint.Enable("github.com/pingcap/tidb/ttl/ttlworker/update-info-schema-cache-interval", fmt.Sprintf("return(%d)", time.Second))
```

Comment:
In general, we will disable the failpoint after testing to avoid flaky test in the next test cases.

Reply:
Fixed. I also fixed it for other tests. Thanks for the advice :heart: